src/eterna/mode/DesignBrowser/SolutionDescBox.ts

```diff
@@ -152,7 +152,10 @@ export default class SolutionDescBox extends GamePanel {
 
             for (let text of this._descriptionAndCommentsTexts) {
                 text.style.wordWrapWidth = this._width - 40;
-                text.dirty = true;
+                // AMW: text.dirty no longer exists.
+                // We can see if this is needed. Maybe now it's automated for
+                // style changes.
+                // text.dirty = true;
```

Comment:
```suggestion
```

According to http://pixijs.download/v5.2.0/docs/PIXI.Text.html#style this is indeed the case